mmcv/ops/csrc/pytorch/mlu/focal_loss_sigmoid_mlu.cpp

```diff
@@ -0,0 +1,165 @@
+/*************************************************************************
+ * Copyright (C) 2021 Cambricon.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *************************************************************************/
+#include "pytorch_mlu_helper.hpp"
+#include <string>
+#include <vector>
+
+#define NFU_ALIGN_SIZE 128
```

Comment:
Can be placed in `pytorch_mlu_helper.hpp` so different ops can share the value.

Reply:
yse, It would be better that we put it in `pytorch_mlu_helper.hpp`